src/unittest/test_collision.cpp

```diff
@@ -16,15 +20,13 @@ class TestCollision : public TestBase {
 
 	void runTests(IGameDef *gamedef);
 
-	void testAxisAlignedCollision();
 	void testCollisionMoveSimple(IGameDef *gamedef);
```

Comment:
I think we shouldn't do partial migrations so that catch and regular tests are mixed in a single file.
Or is there no solution to pass `gamedef` to tests yet?

Reply:
There isn't, yet, but maybe this is the right time to make that solution.

Reply:
I will work on a patch for that, and then we can decide whether to do that here or separately.